docs/components/TextInputView.jsx

```diff
@@ -58,6 +58,8 @@ export default class TextInputView extends Component {
                   console.log("mouseover!", e);
                 }}
                 size={this.state.size}
+                showButtonText="🐵"
+                hideButtonText="🙈"
```

Comment:
As much as I love these choices :smile:, I might suggest leaving the props documented as you have but using the defaults in the actual displayed example

Reply:
Oh wops, I didn't mean to commit this.